src/layout/index.js

```diff
@@ -1,39 +1,65 @@
-import Awaiter from './Awaiter'
 import { Flex } from './Flex'
 import { Grid } from './Grid'
+import { Tag, TagsInput } from './TagsInput'
+import Awaiter from './Awaiter'
+import BarChart from './BarChart'
+import Checkbox from './Checkbox'
+import CheckButton from './CheckButton'
+import Dynamic from './Dynamic'
+import ExpandableTable from './ExpandableTable'
+import LensInput from './LensInput'
 import Modal from './Modal'
-import Popover from './Popover'
-import SpacedList from './SpacedList'
-import TextHighlight from './TextHighlight'
 import ModalPicker from './ModalPicker'
 import NestedPicker from './NestedPicker'
-import Dynamic from './Dynamic'
-import BarChart from './BarChart'
-import { Tag, TagsInput } from './TagsInput'
-import LensInput from './LensInput'
-import ExpandableTable from './ExpandableTable'
+import Popover from './Popover'
 import Portal from './Portal'
-import CheckButton from './CheckButton'
+import SpacedList from './SpacedList'
 import StepsAccordion, { AccordionStep } from './StepsAccordion'
+import TextHighlight from './TextHighlight'
 
-export {
+export let theme = {
+  AccordionStep,
   Awaiter,
+  BarChart,
+  Checkbox,
+  CheckButton,
+  Dynamic,
+  ExpandableTable,
   Flex,
   Grid,
+  LensInput,
   Modal,
+  ModalPicker,
+  NestedPicker,
+  // FieldPicker: NestedPicker,
```

Comment:
This is evidence of a bug where nested themes don't work when the name a component is given by `withTheme` (or `ThemeConsumer`) is different from its key in the theme.

Uncomment this to see the Blueberry picker break, even though `FieldPicker` component still defaults to `NestedPicker`.